app/page-partials/my-dashboard/ClientRoles.tsx

```diff
@@ -228,7 +249,7 @@ const ClientRoles = ({ selectedRequest, alert }: Props) => {
     leftContent = (
       <tbody>
         <tr>
-          <td colSpan={2}>You do not have any roles created yet.</td>
+          <td colSpan={2}>You do not have any roles found.</td>
```

Comment:
This doesn't really sound right to me.  "No roles found" is probably more clear

Reply:
right, just updated the text.